test/feeder_ex_test.exs

```diff
@@ -30,6 +30,11 @@ defmodule FeederExTest do
 
   end
 
+  test "parse! returns just the parsed results on success" do
+    parsed_feed = FeederEx.parse!(File.read!(@sample_file))
+    assert parsed_feed.title == "Liftoff News"
+  end
```

Comment:
`bin_feed = File.read!(@sample_file) |> String.replace("</xml>", "")` seems to work.

i think you discovered a bug in your other case, though. we should catch `feed` being nil in `def event(:endFeed, {feed, entries})` and handle that better.